src/Action/SoftDeleteAction.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace CottaCush\Yii2\Action;
+
+use CottaCush\Yii2\Constants\Messages;
+use CottaCush\Yii2\Controller\BaseController;
+use CottaCush\Yii2\Model\BaseModel;
+use yii\db\IntegrityException;
+
+class SoftDeleteAction extends DeleteAction
```

Comment:
How come you extend DeleteAction but didn't use it at the top

Reply:
Since they are in the same namespace